packages/@vuepress/theme-default/src/client/components/global/CodeGroup.ts

```diff
@@ -63,12 +64,37 @@ export default defineComponent({
                 h(
                   'button',
                   {
+                    ref: (element) => {
+                      if (element) {
+                        tabRefs.value[i] = element as HTMLButtonElement
+                      }
+                    },
                     class: `code-group__nav-tab${
                       i === activeIndex.value
                         ? ' code-group__nav-tab-active'
                         : ''
                     }`,
+                    ariaPressed: i === activeIndex.value,
+                    ariaExpanded: i === activeIndex.value,
                     onClick: () => (activeIndex.value = i),
+                    onKeydown: (event: KeyboardEvent) => {
+                      if (event.key === ' ' || event.key === 'Enter') {
+                        event.preventDefault()
+                        activeIndex.value = i
+                      } else if (event.key === 'ArrowRight') {
+                        event.preventDefault()
+                        if (i + 1 < items.length) {
+                          activeIndex.value = i + 1
+                          tabRefs.value[i + 1].focus()
+                        }
+                      } else if (event.key === 'ArrowLeft') {
+                        event.preventDefault()
+                        if (i - 1 >= 0) {
+                          activeIndex.value = i - 1
+                          tabRefs.value[i - 1].focus()
+                        }
```

Comment:
We could allow navigating from the last item to the first item directly, and vice versa.

Reply:
In addtion, we'd better extract those navigation methods to the setup function.

For example:

```ts
const activeIndex = ref(-1)
const tabRefs = ref<HTMLButtonElement[]>([])
const activateNext = () => {
  // ...
}
const activatePrev = () => {
  // ...
}
```